administrator-guides/integrations/github/README.md

```diff
@@ -255,9 +255,7 @@ This script will generate notifications for the following repository events:
 
 ![Screenshot of messages generated by GitHub integration script 2](github-2.png)
 
-```javascript
-/* exported Script */
-
+```/* exported Script */
```

Comment:
Why was this changed? The docs works better when we define the programming language

Reply:
Ok i checked your way and it does work better as you mentioned, hence reverting to what you had written.